CHANGELOG.md

```diff
@@ -20,7 +20,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * `ui.default-command` now accepts multiple string arguments, for more complex
   default `jj` commands.
 
-* Graph node symbols are now configurable via `ui.graph.default_node` and `ui.graph.elided_node`.
+* Graph node symbols are now configurable via templates
+  * `templates.log_graph_node_commit`
+  * `templates.log_graph_node_op`
+  * `templates.log_graph_node_elided`
```

Comment:
nit: maybe better to name them as `log_graph_node`, `op_log_graph_node`, and `log_graph_node_elided`. The text parts are `log` and `op_log` respectively.

Reply:
Maybe graph could even be left out if the emphasis is on the command
```
log_node
log_node_elided
op_log_node
```

Reply:
I like the former (assuming `_elided` will eventually be combined into `log_node`.)

>  I guess one possible confusion with the first one is that the commit and elided ones also apply to the obslog.

Yep, but that's consistent with the `templates.log` text template. We might want separate `templates.obslog*` definitions, but I don't know.

Reply:
Done.